server/viame_tasks/utils.py

```diff
@@ -6,6 +7,9 @@
 
 from girder_worker.task import Task
 
+# TODO: Move viame_server.constants into a shared area like viame_utils to share constants
+validVideoFormats = [".mp4", ".avi", ".mov", ".mpg"]
```

Comment:
Might as well make it a set instead of a list so that when the refactor happens, it's a drop-in replacement.